database/migrations/20220512210542_master-changes.sql

```diff
@@ -0,0 +1,28 @@
+DROP INDEX media_excl_ep_idx;
+
+DROP VIEW media;
+CREATE VIEW media AS
+SELECT _tblmedia.*, pp.local_path as poster_path, bp.local_path as backdrop_path
+FROM _tblmedia
+LEFT OUTER JOIN assets pp ON _tblmedia.poster = pp.id
+LEFT OUTER JOIN assets bp ON _tblmedia.backdrop = bp.id;
+CREATE TRIGGER media_delete
+INSTEAD OF DELETE ON media
+BEGIN
+    DELETE FROM _tblmedia WHERE _tblmedia.id = old.id;
+END;
```

Comment:
This is a bit unreadable, we should add some blank lines in between sql statements. I think we should stick to 1 blank line in between statements.

Reply:
Better?

Reply:
yep, looks a lot better.